src/dry-package.ts

```diff
@@ -111,4 +112,48 @@ export class DryPackage {
     public get content(): WeakDryPackageContent {
         return JSON.parse(JSON.stringify(this._content));
     }
+
+    /**
+     * This method will resolve inherited version of dependencies
+     * and then delete provided version configuration properties
+     */
+    private resolveInheritedDependencies(): void {
+        let dependencies = this._content.dependencies;
+        let dependenciesMgmt = this._content.dependenciesManagement;
+        this.resolveInheritance(dependencies, dependenciesMgmt);
+
+        dependencies = this._content.devDependencies;
+        dependenciesMgmt = this._content.devDependenciesManagement;
+        this.resolveInheritance(dependencies, dependenciesMgmt);
+
+        delete this._content.dependenciesManagement;
+        delete this._content.devDependenciesManagement;
+    }
+
+    /**
+     * This method will replace any value equals to "inherit" provided
+     * in dependencies parameter by the value of the same key provided
+     * in dependenciesManagement parameter
+     * @param {any} dependencies object containing a list of key/value
+     * @param {any} dependenciesManagement object containing a list of key/value
+     */
+    private resolveInheritance(dependencies: { [s: string]: string; }, dependenciesManagement: { [s: string]: string; }): void {
+        if (dependencies && dependenciesManagement) {
+            for (const key in dependencies) {
+                if (dependencies.hasOwnProperty(key)) {
+                    const inherit = 'inherit'.toUpperCase() === dependencies[key].toUpperCase();
+                    if (inherit) {
+                        const inheritedVersion = dependenciesManagement[key];
+
+                        if (inheritedVersion) {
+                            dependencies[key] = inheritedVersion;
+                        } else {
+                            const message = 'Package ' + key + ' must inherit a version but none are provided!';
+                            throw message;
```

Comment:
message variable is redundant.
Throwing a `new Error('foo')` is a best practice.

Reply:
Ok, done